internal/integtest/setup.go

```diff
@@ -46,6 +47,7 @@ import (
 var (
 	K8sClient client.Client
 	testEnv   *envtest.Environment
+	cancel    context.CancelFunc
```

Comment:
Maybe give this a slightly more descriptive name, e.g. `testEnvCancelFn`? I know that Go prefers short variable names but this seems a bit too terse and made the code a bit hard to understand IMO.

Reply:
I agree with renaming the variable, but I think `k8sManagerCancelFn` is more correct, isn't it?

Reply:
I have renamed it as suggested, but please let me know if you have changed your mind. 😉